app/pods/search/newsletter-infos/route.js

```diff
@@ -0,0 +1,151 @@
+/* eslint-disable class-methods-use-this */
+import Route from '@ember/routing/route';
+import { isEmpty, isPresent } from '@ember/utils';
+import moment from 'moment';
+import search from 'frontend-kaleidos/utils/mu-search';
+import Snapshot from 'frontend-kaleidos/utils/snapshot';
+import { inject as service } from '@ember/service';
+
+export default class AgendaitemSearchRoute extends Route {
+  @service metrics;
+  queryParams = {
+    page: {
+      refreshModel: true,
+      as: 'pagina',
+    },
+    size: {
+      refreshModel: true,
+      as: 'aantal',
+    },
+    sort: {
+      refreshModel: true,
+      as: 'sorteer',
+    },
+  };
+
+  textSearchFields = Object.freeze([
+    'title',
+    'subTitle',
+    'data.content',
```

Comment:
Is searching within related documents (`data.content`) explicitly within scope of this feature? As already stated in https://github.com/kanselarij-vlaanderen/kaleidos-project/pull/235/files#r721199957 , this data-point isn't consistent throughout the whole history of the database. Moreover, I believe that adding full text search in related documents would negatively impact result accuracy, especially when considering that the broadness of full-text-search enabled searches not being considered specific enough for kort-bestek was a trigger for this feature request. I propose to remove full-text search from this feature at least for now.

Reply:
is removed